phf/src/lib.rs

```diff
@@ -5,10 +5,20 @@
 //! the documentation of those crates for more details.
 #![doc(html_root_url="https://sfackler.github.io/rust-phf/doc/v0.7.5")]
 #![warn(missing_docs)]
+#![cfg_attr(feature = "core", feature(no_std))]
+#![cfg_attr(feature = "core", no_std)]
 
 extern crate debug_builders;
 extern crate phf_shared;
 
+#[cfg(feature = "core")]
+mod std {
```

Comment:
Are you referring to the cfg attributes? Yes they are still necessary, otherwise you will get name clashed with std:
src/lib.rs:14:1: 19:2 error: the name `std` conflicts with an external crate that has been imported into this module [E0260]

AFAIK the only thing that is no longer required is the explicit mention of extern crate core;
RFC on no_std stabilization: https://github.com/rust-lang/rfcs/blob/master/text/1184-stabilize-no_std.md